geniza/corpus/admin.py

```diff
@@ -12,8 +15,15 @@ class CollectionAdmin(admin.ModelAdmin):
 
 @admin.register(LanguageScript)
 class LanguageScriptAdmin(admin.ModelAdmin):
-    list_display = ('language', 'script', 'display_name')
-
+    list_display = ('language', 'script', 'display_name', 'usage_count')
+
+    def usage_count(self, obj):
```

Comment:
This label seems a little odd to me. What about documents? Display label could be '# documents'